TodoList/src/app/screens/ToDoList/styles.js

```diff
@@ -1,10 +1,12 @@
-import { StyleSheet } from 'react-native'
+import { StyleSheet } from 'react-native';
 
 export const styles = StyleSheet.create({
   container: {
-    flex: 1
+    flex: 1,
   },
   divider: {
-    height: 1
-  }
-})
+    height: 1,
```

Comment:
You have an extra `,` (but you don't need this style anyway :stuck_out_tongue: )